cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/src/bond0.nmconnection

```diff
@@ -0,0 +1,17 @@
+[connection]
+id=bond0
+type=bond
+interface-name=bond0
+autoconnect=true
+connection.autoconnect-slaves=1
+
+[ethernet]
+mtu=9000
+
+[bond]
+mode=balance-rr
+miimon=140
```

Comment:
what is this pair specifying?

Reply:
I've already covered the `mode` parameter; the `miimon` parameter specifies the link monitoring interval. From [these docs](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-using_channel_bonding):

> `miimon=time_in_milliseconds`
>
> Specifies (in milliseconds) how often MII link monitoring occurs. This is useful if high availability is required because MII is used to verify that the NIC is active. To verify that the driver for a particular NIC supports the MII tool, type the following command as root:

I'm using this particular value because that's what @jtriley used when doing the test install on these systems.